CHANGELOG.md

```diff
@@ -1,17 +1,26 @@
 # Mobly Release History
 
 
+## Mobly Release 1.7.5: Dependency Cleanup
+
+### Fixes
+* Only install test dependencies when running the unit tests.
+* Limit `CallbackHandler.waitForEvent` to the max rpc timeout.
```

Comment:
this is actually the opposite...
the change was to remove the max timeout limit for that API